webapp/web/configuration_form.css

```
@@ -2,64 +2,103 @@
     display: grid;
     grid-template-rows: auto;
     grid-gap: 20px;
-    border: 1px solid #ddd;
-    border-radius: .5rem;
     padding: 20px;
```

It looks far too much spread out at the moment. Let's centre the contents and set a width:
```suggestion
    padding: 20px;
    max-width: 700px;
    margin: 0 auto;
```